.github/workflows/gradle.yml

```
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+
+    - name: Create application.properties
+      run: echo "${{ secrets.APPLICATION }}" > ./src/main/resources/application.yml
```

제가 알기로 바로 넣게 되면 따옴표 같은 게 있으면 빌드 에러가 되서 Base64로 인코딩 해서 넣고 디코딩 하면 좋을 것 같아요..!
```suggestion
      run: echo "${{ secrets.APPLICATION }}" | base64 -d > ./src/main/resources/application.yml
```